src/session_handle.cpp

```diff
@@ -92,12 +93,12 @@ namespace libtorrent {
 	{
 		std::shared_ptr<session_impl> s = m_impl.lock();
 		if (!s) aux::throw_ex<system_error>(errors::invalid_session_handle);
-		dispatch(s->get_context(), [=]() mutable
+		dispatch(s->get_context(), std::bind([s, f](auto&&... args) mutable
```

Comment:
this is to emulate `[s, f, args = std::move(a)]`, or do you always need `std::bind()` since `a` is a parameter pack?

Reply:
This is to emulate `[s, f, ...args = std::forward<Args>(a)]` since explicit capturing of parameters pack is available only since c++20.

Reply:
@arvidn 
So what are you going to do about this PR?

Reply:
sorry for the delay. I'll take a deeper look today.